jcl/src/java.base/share/classes/com/ibm/oti/util/Util.java

```diff
@@ -280,11 +300,11 @@ public static void printStackTraceElement(StackTraceElement e, Object elementSou
 			}
 		}
 		appendTo(buf, "/"); //$NON-NLS-1$
+	} else if (extended) {
+		appendTo(buf, "app//"); //$NON-NLS-1$
```

Comment:
I don't think this is correct. According to my testing, `app//` should only be printed when there isn't a package name.

Reply:
What version of Java are you testing with?

Reply:
I re-tested and even with the package I see the `app//` prefix today.

Reply:
I am testing with Java 11.

Reply:
> #510: java -version
> openjdk version "11.0.2" 2019-01-15
> OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.2+9)
> OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.2+9, mixed mode)
> 
> #513: java test.DumpThreads
> "main" prio=5 Id=1 RUNNABLE
> 	at java.management@11.0.2/sun.management.ThreadImpl.dumpThreads0(Native Method)
> 	at java.management@11.0.2/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:502)
> 	at java.management@11.0.2/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:490)
> 	at app//test.DumpThreads.main(DumpThreads.java:7)

Note that running the `.java` file directly suppresses the `app//` prefix.